.github/workflows/CI.yml

```diff
@@ -47,6 +47,6 @@ jobs:
           git diff --exit-code
       - name: Run pytest
         run: |
-          python -m pytest
-          mpirun -n 1 python -m pytest --with-mpi
-          mpirun -n 2 python -m pytest --with-mpi
+          python -m pytest -rfEP
```

Comment:
@streeve what does "-rfEP" do?

Reply:
Prints failing, errored, and passing (default does not include passing)

Reply:
Fancy!